scipy/stats/_binomtest.py

```diff
@@ -0,0 +1,314 @@
+        Indicates the alternative hypothesis specified in the input
+        to `binomtest`.  It will be one of ``'two-sided'``, ``'greater'``,
+        or ``'less'``.
+    pvalue : float
+        The p-value of the hypothesis test.
+    proportion_estimate : float
+        The estimate of the proportion of successes.
+
+    """
+    def __init__(self, k, n, alternative, pvalue, proportion_estimate):
+        self.k = k
+        self.n = n
+        self.alternative = alternative
+        self.proportion_estimate = proportion_estimate
+        self.pvalue = pvalue
+
+    def __repr__(self):
+        s = ("BinomTestResult("
+             f"k={self.k}, "
+             f"n={self.n}, "
+             f"alternative={self.alternative!r}, "
+             f"proportion_estimate={self.proportion_estimate}, "
+             f"pvalue={self.pvalue})")
+        return s
+
+    def proportion_ci(self, confidence_level=0.95, method='exact'):
+        """
+        Compute the confidence interval for the estimated proportion.
+
+        Parameters
+        ----------
+        confidence_level : float, optional
+            Confidence level for the computed confidence interval
+            of the estimated proportion. Default is 0.95.
+        method : {'exact', 'wilson', 'wilsoncc'}, optional
+            Selects the method used to compute the confidence interval
+            for the estimate of the proportion:
+
+            'exact' :
+                Use the Clopper-Pearson exact method [1]_.
+            'wilson' :
+                Wilson's method, without continuity correction ([2]_, [3]_).
+            'wilsoncc' :
+                Wilson's method, with continuity correction ([2]_, [3]_).
+
+            Default is ``'exact'``.
+
+        Returns
+        -------
+        ci : namedtuple with fields ``low`` and ``high``
+            Contains the lower and upper bounds of the confidence interval.
+
+        References
+        ----------
+        .. [1] C. J. Clopper and E. S. Pearson, The use of confidence or
+               fiducial limits illustrated in the case of the binomial,
+               Biometrika, Vol. 26, No. 4, pp 404-413 (Dec. 1934).
+        .. [2] E. B. Wilson, Probable inference, the law of succession, and
+               statistical inference, J. Amer. Stat. Assoc., 22, pp 209-212
+               (1927).
+        .. [3] Robert G. Newcombe, Two-sided confidence intervals for the
+               single proportion: comparison of seven methods, Statistics
+               in Medicine, 17, pp 857-872 (1998).
+
+        Examples
+        --------
+        >>> from scipy.stats import binomtest
+        >>> result = binomtest(k=7, n=50, p=0.1)
+        >>> result.proportion_estimate
+        0.14
+        >>> result.proportion_ci
+        ConfidenceInterval(low=0.04246878737883377, high=0.30910696596481874)
+        """
+        if method not in ('exact', 'wilson', 'wilsoncc'):
+            raise ValueError("method must be one of 'exact', 'wilson' or "
```

Comment:
Maybe test this to make codecov happy.

Reply:
Hey Matt--just random drive-by comment; if codecov is actually working well now that's great, but the patch diff could very well be failing because we don't have a common base commit available for codecov just yet: https://github.com/scipy/scipy/issues/13077

Just so you are aware--can double check by clicking through and checking the "diff" tab codecov is using--may be nonsense at the moment.

Reply:
I added a test that checks for a ValueError when an invalid method is given.

Reply:
mmmm shrimp.